### DISCUSSION TITLE
User Personas for the FOSS United Profiles

### DISCUSSION THREAD

**Discussion by rahulporuri** [fossunited/fossunited#454] [General]
This discussion outlines the User Personas for the FOSS United Profiles. The User Personas should help us identify usecases for FOSS United Profiles. Read https://www.nngroup.com/articles/persona/ for a better understanding of the what and the why of User Personas.

* Aakansha leads a FOSS United city chapter. She needs a FOSS United Profile to be listed as a lead on the city chapter page and to manage the chapter page e.g. create monthly meetup event page.
* Arun is a volunteer with a FOSS United city chapter. He needs a FOSS United Profile to be listed as a volunteer on the city chapter page and to manage to chapter page e.g. create monthly meetup event page.
* Balu is interested in speaking at a FOSS United city chapter. He needs a FOSS United Profile to submit a talk/workshop proposal on the city chapter page
* Rahul is interested in attending a monthly meetup at a FOSS United city chapter. He can choose to sign up for a FOSS United Profile and RSVP for the meetup. He could also sign up anonymously but we will be able to recognize regular attendees if they RSVP using a FOSS United Profile e.g. by giving them brownie points or badges for being regulars
* Vanya works at a Venue Partner for a monthly meetup. She would like to see some information about submitted proposals and RSVPs to gauge interest in the meetup from the community. She isn't an active member of the FOSS United community yet so she doesn't have a FOSS United Profile at the moment

I feel like I am missing a few User Personas but I can't put my finger on what they are.

**Comment by harshtandiya** [comment]
- Mangesh is a CFP review volunteer at FOSS United, he needs a FOSS User Profile + a role assigned to him as a "CFP Reviewer" to be able to submit his reviews
- Anoop is a hackathon localhost organizer, he needs to have a FOSS User Profile + a role assigned as `Localhost Organizer` to be able to go through the localhost attendee requests.
- Vishal wants to attend a hackathon taking place on platform, he needs to have a FOSS User Profile to signup and create teams
- Ruchika wants to buy a ticket to attend a conference such as IndiaFOSS, she can either do it anonymously without creating an account or create an account so that the ticket will always be linked to her user in the database

Adding to the Chapter members, they need to have a FOSS User Profile + a role `Chapter Team Member` or `Chapter Lead` assigned to them in order to use the dashboard to manage their chapter.

**Reply by rahulporuri** [reply]
Note that the Roles that you mentioned above are an implementation detail and User Personas are abstract concepts. The Roles are derived from the needs/actions of the User Personas. For example, here is how I would write the "Mangesh" User Persona - "Mangesh reviews talk/workshop Proposal for monthly meetups and he needs a FOSS United Profile to be able to provide review comments." -> in order to perform this action, we include a "CFP Reviewer" role to the FOSS United Profile implementation, which determines whether or not someone can be a "CFP Reviewer".

> Adding to the Chapter members, they need to have a FOSS User Profile + a role `Chapter Team Member` or `Chapter Lead` assigned to them in order to use the dashboard to manage their chapter.

I've updated the Aakansha and Arun User Personas to mention that they FOSS United Profile will determine if they can manage their chapter.

**Comment by mngshm** [comment]
Some of the user personas I could think of

1. Ayush (from Mumbai Chapter) is a designer and he needs to see the CFP Speaker data to design the social media content. He needs a FOSS Profile to be a able to be a part of a chapter and see the insights on selected talks for the event. 
2. Ansh is a FOSS Club lead and he needs a FOSS Profile to be able to manage events and his club page and members.

**Comment by rahulporuri** [comment]
I came across these incredibly detailed Learner Personas https://third-bit.com/ideas/learner-personas/ which should help us expand and establish the various Personas that we expect to use the Platform
